configuration/compatibility-testing/test-rp-msa-4.2.1-901.yml

```diff
@@ -81,27 +81,4 @@ encryptionKeys:
 returnStackTraceInErrorResponse: true
 
 europeanIdentity:
-  enabled: true
```

Comment:
This config is used when spinning up an old version of the MSA for the backwards compatibility tests. I think you might have to leave it as is.

Reply:
Good spot, will restore!